README.md

```diff
@@ -61,15 +61,18 @@ Read more about [Verify Pacts](https://github.com/realestate-com-au/pact/wiki/Ve
 ```js
 var pact = require('@pact-foundation/pact-node');
 var opts = {
-	providerBaseUrl: <String>,       // Running API provider host endpoint. Required.
-	pactUrls: <Array>,               // Array of local Pact file paths or Pact Broker URLs (http based). Required.
-	providerStatesUrl: <String>,     // URL to fetch the provider states for the given provider API. Optional.
-	providerStatesSetupUrl: <String>, // URL to send PUT requests to setup a given provider state. Optional.
-	pactBrokerUsername: <String>,    // Username for Pact Broker basic authentication. Optional
-	pactBrokerPassword: <String>,    // Password for Pact Broker basic authentication. Optional
+	providerBaseUrl: <String>,           // Running API provider host endpoint. Required.
+	pactBrokerUrl: <String>              // URL of the Pact Broker to retrieve pacts from. Required if not using pactUrls.
+	provider: <String>                   // Name of the Provider. Required.
+	tags: <Array>                        // Array of tags, used to filter pacts from the Broker. Optional.
```

Comment:
Huh, weird.  It was working before.  Must of been a refactor that didn't go through or something.  I need to get typescript going for it to fix some of these easily identifiable issues.  But yeah, if we can get it working again, that'd be great.  I think comma delimited would be good enough.